### PR TITLE
new package: patch 2.7

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

**Comment by a-schaefers** [comment]
fixed

File: community/patch/build [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+export CC="gcc -static"
+
+./configure \
+    --prefix=/usr \
+    --program-prefix=g
+
+make DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/patch/build]: Missing standalone `make`?

Comment (a-schaefers) [reply community/patch/build]: i did not know we preferred that. will do

Comment (a-schaefers) [reply community/patch/build]: I did notice once in a blue moon a package will fail without the standalone make. *cough* iptables

Comment (a-schaefers) [reply community/patch/build]: done

**Pull request merged**